src/System.Diagnostics.DiagnosticSource/ref/System.Diagnostics.DiagnosticSource.cs

```diff
@@ -7,26 +7,26 @@
 
 namespace System.Diagnostics
 {
-    public partial class DiagnosticListener : System.Diagnostics.DiagnosticSource, System.IDisposable, System.IObservable<System.Collections.Generic.KeyValuePair<string, object>>
+    public partial class DiagnosticListener : System.Diagnostics.DiagnosticSource, System.IDisposable, System.IObservable<System.Collections.Generic.KeyValuePair<string?, object?>>
     {
-        public DiagnosticListener(string name) { }
+        public DiagnosticListener(string? name) { }
         public static System.IObservable<System.Diagnostics.DiagnosticListener> AllListeners { get { throw null; } }
-        public string Name { get { throw null; } }
+        public string? Name { get { throw null; } }
         public virtual void Dispose() { }
         public bool IsEnabled() { throw null; }
         public override bool IsEnabled(string name) { throw null; }
-        public override bool IsEnabled(string name, object arg1, object arg2 = null) { throw null; }
-        public virtual System.IDisposable Subscribe(System.IObserver<System.Collections.Generic.KeyValuePair<string, object>> observer) { throw null; }
-        public virtual System.IDisposable Subscribe(System.IObserver<System.Collections.Generic.KeyValuePair<string, object>> observer, System.Func<string, object, object, bool> isEnabled) { throw null; }
-        public virtual System.IDisposable Subscribe(System.IObserver<System.Collections.Generic.KeyValuePair<string, object>> observer, System.Predicate<string> isEnabled) { throw null; }
-        public override string ToString() { throw null; }
-        public override void Write(string name, object value) { }
+        public override bool IsEnabled(string name, object? arg1, object? arg2 = null) { throw null; }
+        public virtual System.IDisposable Subscribe(System.IObserver<System.Collections.Generic.KeyValuePair<string?, object?>> observer) { throw null; }
+        public virtual System.IDisposable Subscribe(System.IObserver<System.Collections.Generic.KeyValuePair<string?, object?>> observer, System.Func<string, object?, object?, bool>? isEnabled) { throw null; }
+        public virtual System.IDisposable Subscribe(System.IObserver<System.Collections.Generic.KeyValuePair<string?, object?>> observer, System.Predicate<string>? isEnabled) { throw null; }
+        public override string? ToString() { throw null; }
+        public override void Write(string? name, object? value) { }
```

Comment:
If we change the name in the Subscribe function to be non-nullable then this one should be non-nullable as well.